lib/animina_web/live/profile_live.ex

```diff
@@ -165,11 +170,11 @@ defmodule AniminaWeb.ProfileLive do
         </div>
 
         <div class="mt-8 space-y-4">
-          <h2 class="font-bold dark:text-white text-xl">My White Flags</h2>
+          <h2 class="font-bold text-xl">My White Flags</h2>
```

Comment:
Always use some form of getext for those texts. Otherwise we can not translate it.